dbt_semantic_interfaces/type_enums/date_part.py

```diff
@@ -0,0 +1,44 @@
+from enum import Enum
+from typing import List
+
+from dbt_semantic_interfaces.enum_extension import assert_values_exhausted
+from dbt_semantic_interfaces.type_enums.time_granularity import TimeGranularity
+
+
+class DatePart(Enum):
+    """Date parts able to be extracted from a time dimension.
+
+    TODO: add support for hour, minute, second once those granularities are available
+    """
+
+    YEAR = "year"
+    QUARTER = "quarter"
+    MONTH = "month"
+    WEEK = "week"
```

Comment:
We no longer support WEEK, and I'll be merging that change to MetricFlow tonight or tomorrow. https://github.com/dbt-labs/metricflow/pull/812

Reply:
➕